vaxrank/epitope_prediction.py

```diff
@@ -47,7 +47,7 @@ def logistic_epitope_score(
             self,
             midpoint=350.0,
             width=150.0,
-            ic50_cutoff=2000.0):
+            ic50_cutoff=5000.0):
```

Comment:
Maybe add a TODO comment about moving these default values into the CLI as arguments?

Reply:
done